guides/common/assembly_using-puppet-for-configuration-management.adoc

```diff
@@ -0,0 +1,45 @@
+ifdef::context[:parent-context: {context}]
+
+[id="using-puppet-for-configuration-management_{context}"]
```

Comment:
AFAIK - this _{context} should be added to each ID of each module for maximum flexibility for future reuse. 
Not going to block the PR but it might be good practice

Reply:
Added `_{context}` and rebased to master.